src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -36,28 +36,6 @@ static constexpr std::array<std::wstring_view, 11> DefaultFeatures{
 
 namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 {
-    bool Font::HasPowerlineCharacters()
-    {
-        if (!_hasPowerlineCharacters.has_value())
-        {
-            try
-            {
-                winrt::com_ptr<IDWriteFont> font;
-                THROW_IF_FAILED(_family->GetFont(0, font.put()));
-                BOOL exists{};
-                // We're actually checking for the "Extended" PowerLine glyph set.
-                // They're more fun.
-                THROW_IF_FAILED(font->HasCharacter(0xE0B6, &exists));
-                _hasPowerlineCharacters = (exists == TRUE);
-            }
-            catch (...)
-            {
-                _hasPowerlineCharacters = false;
-            }
-        }
-        return _hasPowerlineCharacters.value_or(false);
-    }
```

Comment:
`Font::HasPowerlineCharacters()` was moved to `AppearanceViewModel::HasPowerlineCharacters()`, because "has" is now a function of >1 fonts (= can't be attached to a single `Font` instance). The detection itself went into `_refreshFontFaceDependents` below.